packages/twenty-front/src/modules/object-record/object-filter-dropdown/hooks/useSetFilterDefinitionUsedInDropdownInScope.ts

```diff
@@ -0,0 +1,22 @@
+import { useRecoilCallback } from 'recoil';
+import { filterDefinitionUsedInDropdownComponentState } from '../states/filterDefinitionUsedInDropdownComponentState';
+import { FilterDefinition } from '../types/FilterDefinition';
+import { extractComponentState } from '@/ui/utilities/state/component-state/utils/extractComponentState';
+
+export const useSetFilterDefinitionUsedInDropdownInScope = () => {
```

Comment:
It's not a common pattern but I'll refactor this part soon so let's leave it like that.